agent/src/main.cpp

```diff
@@ -216,11 +216,15 @@ int main(int argc, char **argv)
         std::shared_ptr<Config> config = std::make_shared<Config>(std::move(configSources),
                                                                   ConfigDefinition::mandatory(),
                                                                   ConfigDependencies());
-        std::shared_ptr<SoftwareContainerFactory> factory = std::shared_ptr<SoftwareContainerFactory> (new SoftwareContainerFactory());
-        std::shared_ptr<ContainerUtilityInterface> utility = std::shared_ptr<ContainerUtilityInterface> (new ContainerUtilityInterface());
+        std::shared_ptr<SoftwareContainerFactory> factory =
+            std::shared_ptr<SoftwareContainerFactory> (new SoftwareContainerFactory());
+        std::shared_ptr<ContainerUtilityInterface> utility =
+            std::shared_ptr<ContainerUtilityInterface> (new ContainerUtilityInterface());
 
         ::softwarecontainer::SoftwareContainerAgent agent(mainContext, config, factory, utility);
-        std::unique_ptr<SoftwareContainerAgentAdaptor> adaptor(new SoftwareContainerAgentAdaptor(agent, useSessionBus));
+        std::unique_ptr<SoftwareContainerAgentAdaptor> adaptor(
+            new SoftwareContainerAgentAdaptor(ml, agent, useSessionBus)
```

Comment:
Maybe there should be a comment what the agent adaptor needs the mainloop for? The reader can ofc look at the implementation but I'd like to see that already here if I read this.